### PR TITLE
Changelog: Noting 3.15

### PR DESCRIPTION
**Pull request by oleg-nenashev**
@reviewbybees @rysteboe

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,61 @@ This file also provides links to Jenkins versions,
...
+
+Release date: Dec 21, 2017
+
+Enhancements:
+
+* [JENKINS-48133](https://issues.jenkins-ci.org/browse/JENKINS-48133) -
+Channel exceptions now record the channel name and other information when possible
+* [PR #210](https://github.com/jenkinsci/remoting/pull/210) - 
+Allow disabling HTTPs certificate validation of JNLP endpoint when starting Remoting
+  * **WARNING**: This option undermines the HTTPs securiyt and opens the connection to MiTM attacks
```
Comment (rysteboe) [inline CHANGELOG.md]: securiyt -> security

**Comment by oleg-nenashev** [comment]
PR to the core: https://github.com/jenkinsci/jenkins/pull/3145

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,61 @@ This file also provides links to Jenkins versions,
...
+Release date: Dec 22, 2017
+
+Enhancements:
+
+* [JENKINS-48133](https://issues.jenkins-ci.org/browse/JENKINS-48133) -
+Channel exceptions now record the channel name and other information when possible
+* [PR #210](https://github.com/jenkinsci/remoting/pull/210) - 
+Allow disabling HTTPs certificate validation of JNLP endpoint when starting Remoting
+  * **WARNING**: This option undermines the HTTPs security and opens the connection to MiTM attacks
+    Use it at your own risk
```
Comment (jglick) [inline CHANGELOG.md]: Missing period (`.`) between sentences.

File: CHANGELOG.md [hunk 3]
```diff
@@ -6,6 +6,61 @@ This file also provides links to Jenkins versions,
...
+* [JENKINS-48055](https://issues.jenkins-ci.org/browse/JENKINS-48055) -
+API: Introduce new `getChannelOrFail()` and `getOpenChannelOrFail()` methods in
+[hudson.remoting.Callable](http://javadoc.jenkins.io/component/remoting/hudson/remoting/Callable.html).
+* [JENKINS-37566](https://issues.jenkins-ci.org/browse/JENKINS-37566) -
+API: `Channel#current()` now explicitly requires checking for `null`.
+* [PR #227](https://github.com/jenkinsci/remoting/pull/227) - 
+API: Deprecate and restrict the [JNLP3-connnect protocol](docs/protocols.md) codebase
```
Comment (jglick) [inline CHANGELOG.md]: typo

File: CHANGELOG.md [hunk 4]
```diff
@@ -6,6 +6,61 @@ This file also provides links to Jenkins versions,
...
+* [JENKINS-48130](https://issues.jenkins-ci.org/browse/JENKINS-48130) -
+Prevent fatal failure of `NIOChannelHub` when an underlying executor service rejects a task execution.
+After the change such failure terminates only a single channel
+  * Affected [protocols](docs/protocols.md): JNLP, JNLP2, CLI and CLI2. JNLP4 is not affected
+  * The change also improves diagnostics of `RejectedExecutionException` in other execution services
+* [JENKINS-37670](https://issues.jenkins-ci.org/browse/JENKINS-37670) -
+Throw the standard `UnsupportedClassVersionError` in `RemoteClassLoader` 
+when the bytecode is not supported.
+* [JENKINS-37566](https://issues.jenkins-ci.org/browse/JENKINS-37566) - 
+Cleanup all issues reported by FindBugs. Notable issues:
```
Comment (jglick) [inline CHANGELOG.md]: `Clean up`

**Pull request merged**